app/services/send_sign_up_email_confirmation.rb

```diff
@@ -83,4 +82,6 @@ def send_pw_reset_request_unconfirmed_user_email(request_id, instructions)
   def handle_multiple_email_address_error
     raise 'sign up user has multiple email address records'
   end
+
+  add_method_tracer(:call, "Custom/#{self.name}/call")
```

Comment:
for consistency with the other files, should we remove these parens?

Reply:
👍🏼